### PR TITLE
[#183342339] Merge and combine latest django-cron changes

### PR DESCRIPTION
**Pull request by GitTiago**
Combine Housekeep authored migrations and tests with latest django-cron changes

### CODE REVIEW

File: .github/workflows/django.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+        django_version: [<3, <4]
```
Comment (horthbynorthwest) [inline .github/workflows/django.yml]: Is there a reason we're not specifying 2.2 here? 🤔

File: .travis.yml [hunk 2]
```diff
@@ -1,29 +1,46 @@
...
+    - python: "2.7"
+      env: DJANGO=1.11.* DJANGO_SETTINGS_MODULE='settings_mysql'
+      services:
+        - mysql
+      before_script:
+        - mysql -e 'create database travis_test;'
+    - python: "3.7"
+      env: DJANGO=1.11.* DJANGO_SETTINGS_MODULE='settings_sqllite'
+    - python: "3.7"
+      env: DJANGO=2.0.* DJANGO_SETTINGS_MODULE='settings_sqllite'
```
Comment (horthbynorthwest) [inline .travis.yml:23]: Presumably we don't need to update this to include every single combination, right? Shouldn't this be the LTS version of 2.2 over 2.0?

Also I notice there doesn't seem to be a Django 3 and postgres settings set up? Should there be?

File: docs/sample_cron_configurations.rst [hunk 3]
```diff
@@ -87,7 +162,7 @@ is set to run on each ``runcrons`` task, and the default process is to email
 all the users specified in the ``ADMINS`` settings list when a job fails more
 than 10 times in a row.
 
-Install required dependencies: ``Django>=1.7.0``, ``django-common>=0.5.1``.
+Install required dependencies: ``Django>=3.2.0``.
```
Comment (horthbynorthwest) [inline docs/sample_cron_configurations.rst]: Does this run the risk of installing a higher version of django than we want?

File: test_requirements.txt [hunk 4]
```diff
@@ -1,5 +1,4 @@
 freezegun==0.2.8
 mock==2.0.0
-mysqlclient==1.3.6
-psycopg2==2.6
+psycopg2==2.9.3
```
Comment (horthbynorthwest) [inline test_requirements.txt:3]: `mysqlclient` gets removed here - but the travis file mentions set up with sql settings - am I missing something here?

**Review by horthbynorthwest — COMMENTED** [review]
Got some questions (which I don't think are about your work, but potentially impact the tests)

Will practically test now and circle back

File: .travis.yml (hunk 2 above)
Comment (GitTiago) [reply .travis.yml:23]: travis tests is not used. It was migrated from travis to Github actions

File: .github/workflows/django.yml (hunk 1 above)
Comment (GitTiago) [reply .github/workflows/django.yml]: not really it's usually equivalent, when you specify <3 it install the latest version under version 3, i.e 2.2.X whatever is in vogue

File: test_requirements.txt (hunk 4 above)
Comment (GitTiago) [reply test_requirements.txt:3]: Travis file is no used unfortunately

**Review by horthbynorthwest — APPROVED** [review]
🦭 🚀

**Pull request merged**